apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -43,7 +46,28 @@ const Auth: React.FC = () => {
 		return null;
 	};
 
-	return <>{getScreen(pathname)}</>;
+	return (
+		<>
+			<main className={styles["auth-container"]}>
+				<section className={styles["form-container"]}>
+					{getScreen(pathname)}
+				</section>
+				<section className={styles["ilustration-container"]}>
+					<img
+						alt="sky blue ilustation"
+						className={styles["img-left"]}
+						src={authIlustrationLeft}
+					/>
+					<h1 className={styles["title"]}>Logo</h1>
+					<img
+						alt="sky blue ilustation"
+						className={styles["img-right"]}
+						src={authIlustrationRight}
```

Comment:
What is illustration?

Reply:
Ilustration is svg picture that combine 2 image beside sign up and sign in form. **The picture is as below**

![auth-ilustration-left](https://github.com/user-attachments/assets/ec4db1bf-75c4-44c2-b8cf-a9059cf231ea)
![auth-ilustration-right](https://github.com/user-attachments/assets/0fb13a4f-219d-414f-9f5e-b6e652921223)